types/lodash/ts3.1/common/object.d.ts

```diff
@@ -1122,6 +1122,10 @@ declare module "../index" {
          * @see _.get
          */
         get<TKey extends keyof T>(path: TKey | [TKey]): ExpChain<T[TKey]>;
+        /**
+         * @see _.get
+         */
+        get<TKey extends keyof T>(path: TKey | [TKey], defaultValue: []): T[TKey] extends any[] ? ExpChain<Exclude<T[TKey], undefined>> : ExpChain<Exclude<T[TKey], undefined> | []>;
```

Comment:
1. You should use `never[]` instead of `[]` since that's the type of `[]`.
2. I don't understand how the conditional type helps. Why do you need to exclude `[]`?

Reply:
I left another comment explaining some of my rationale (and one potential alternative to this fix).  

I'm using a conditional type, because I only want the "empty-array" workaround to apply when `T[TKey]` is actually an array.  If we were to do something like:

```
_.chain({ a: 1 }).get('a', []).value();
```

I'd want to ensure that the return type is still `number | never[]`, rather than `number[]` (which is what it would be without the conditional).